.env

```diff
@@ -0,0 +1,6 @@
+ENVIRONMENT=dev
```

Comment:
I'm guessing we don't want to merge this with all the information filled out -> or we at least want a comment that explains they are placeholders.

Reply:
Adjusted, it should be less confusing like that.